nsxt/resource_nsxt_policy_service.go

```diff
@@ -433,9 +440,28 @@ func resourceNsxtPolicyServiceCreate(d *schema.ResourceData, m interface{}) erro
 
 	// Create the resource using PATCH
 	log.Printf("[INFO] Creating service with ID %s", id)
-	err = client.Patch(id, obj)
-	if err != nil {
-		return handleCreateError("Service", id, err)
+
+	if isPolicyGlobalManager(m) {
+		converter := bindings.NewTypeConverter()
+		converter.SetMode(bindings.REST)
+
+		dataValue, err1 := converter.ConvertToVapi(obj, model.ServiceBindingType())
```

Comment:
I've made a utility for this - I can replace this later.

Reply:
Sure. Thanks.